src/chainparams.cpp

```diff
@@ -163,13 +163,13 @@ class CTestNetParams : public CMainParams {
         nMaxTipAge = 0x7fffffff;
         nPruneAfterHeight = 1000;
 
-        // 1MB max blocks before 1 Aug 2015
-        // Then, if miner consensus: 8MB max, doubling every two years
+        // 1MB max blocks before 1 Mar 2014
+        // Then, if miner consensus: 2MB max, doubling twice over two years
         consensus.nMaxSizePreFork = 1000*1000; // 1MB max pre-fork
-        consensus.nEarliestSizeForkTime = 1438387200; // 1 Aug 2015 00:00:00 UTC
+        consensus.nEarliestSizeForkTime = 1393632000; // 01 Mar 2014 00:00:00 UTC, 2 years ahead of mainnet
```

Comment:
Yes, the change matters. The activation date is used for the scaling formula for determining the max block size at any given time. Using a date in the past allows for testnet to be forked immediately, and gives a blocksize limit that is larger than what you'd have available at the time on mainnet. For example, this allows us to fork testnet in February for testing, and would give us a blocksize limit on testnet of around 3.9 MB or so. It would continue to scale after March, so we can continue to test the exact byte limits if we want to.